src/Features/CSharp/Portable/Diagnostics/Analyzers/CSharpTypeStyleDiagnosticAnalyzerBase.cs

```diff
@@ -11,12 +11,84 @@
 using Microsoft.CodeAnalysis.Diagnostics;
 using Microsoft.CodeAnalysis.Options;
 using Microsoft.CodeAnalysis.Text;
+using static Microsoft.CodeAnalysis.CSharp.Diagnostics.TypeStyle.CSharpTypeStyleDiagnosticAnalyzerBase;
 
 namespace Microsoft.CodeAnalysis.CSharp.Diagnostics.TypeStyle
 {
+    internal abstract partial class CSharpTypeStyleHelper
```

Comment:
should be in its own file.

Reply:
I'll move this after review is done, as I think it simplifies the diff.

---
In reply to: [172284494](https://github.com/dotnet/roslyn/pull/25208#discussion_r172284494) [](ancestors = 172284494)